viewing_party/party.py

```diff
@@ -0,0 +1,170 @@
+# Wave 1
+
+def create_movie(title, genre, rating):
+    create_movie_dict = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    if title and genre and rating:
+      return create_movie_dict 
+    else:
+      return None
+
+def add_to_watched(user_data, movie):
+    watched_list = user_data["watched"]
+    watched_list.append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
```

Comment:
this is a good implementation. You could also do it in one line like `user_data["watchlist"] = [movie]`